### ISSUE TITLE
Custom URL field

### ISSUE THREAD

**Issue by turecross321** [LBPUnion/UnionPatcher#14]
There should be a way to patch an already patched eboot if you enter the ip address the eboot's patched to.

**Comment by AShifter** [comment]
Good idea - would make things easier but it kind of conflicts with #9 - we would need to actually store the locations of each URL in each version of the EBOOT unless we could come up with a better way of detecting that 'something' is a URL.

Right now the preferred method is to ensure that you ALWAYS have a backup of your original EBOOT.bin in the event that you want to connect to a different server - then it's as simple as copying that file and patching it multiple times with different URLs, and copying them to access different servers.

UnionEBOOTManager maybe? Lol.

**Comment by jvyden** [comment]
> Right now the preferred method is to ensure that you ALWAYS have a backup of your original EBOOT.bin in the event that you want to connect to a different server

Agreed. It's already a step in our guides to make a backup of the EBOOT.

**Comment by FridiNaTor1** [comment]
What if there was a config file created, that saved previously used URLs, or that you can put URLs in. First time starting the program would create the config with the official URLs, and you could add your old ones or different ones just into the config file. And UnionPatcher would also write the replacement URL into the config automatically, so you can still use the same eboot for repatching.
